### PR TITLE
Amendments for third delivery.

### PR DESCRIPTION
**Pull request by ailithewing**
This is the branch that I'll teach the first two days of the third delivery from. 

Cata and I have removed the scaling/centring from OLS model completely and removed the comparison of coefficients betweeen OLS and regularised models. We think this is simpler and we can simply compare mean square error. 

We've also taken out the callout boxes on scaling/centring and regularisation from a bayesian perspective.

### CODE REVIEW

File: _episodes/03-regression-regularisation.md [hunk 1]
```diff
@@ -213,7 +240,7 @@ different combinations of slope and intercept accomplish this objective.
 Mathematically, we can write the sum of squared residuals as
 
 $$
-    \sum_{i=1}^N (\hat{y}_i - y_i)^2
+    \sum_{i=1}^N ( y_i-X_i\beta)^2
 $$
 
 where $\hat{y}_i$ is the predicted y value for each input data
```
Comment (Alanocallaghan) [inline _episodes/03-regression-regularisation.md:246]: this shouldn't refer to yhat any more

Comment (catavallejos) [reply _episodes/03-regression-regularisation.md:246]: I agree. This should be 

```suggestion
where $X_i\beta$ is the predicted y value for each input data
```

Comment (catavallejos) [reply _episodes/03-regression-regularisation.md:246]: There is an over-use of notation for $X_i$. In the first equation (line 47) it denotes a specific covariate; here it denotes a specific observation. Too late to consistently change the notation everywhere, but we can fix this prior to the next delivery. @ailithewing could you please add this to the list any other things that you may pick up during delivery?

**Comment by catavallejos** [comment]
Thanks @ailithewing I can merge once you fix the issue with *yhat*

File: _episodes/03-regression-regularisation.md (hunk 1 above)
Comment (ailithewing) [reply _episodes/03-regression-regularisation.md]: Yes I've added it to the list which I'll turn into an issue.

**Comment by catavallejos** [comment]
Somehow this PR is still complaining about conflicts, but I have resolved them and manually merged into master. Therefore, I am closing this PR.

**Pull request closed without merging**